cli/cmd/cluster.go

```diff
@@ -663,38 +669,60 @@ var _clusterExportCmd = &cobra.Command{
 	},
 }
 
-func cmdInfo(awsClient *aws.Client, accessConfig *clusterconfig.AccessConfig, disallowPrompt bool) {
-	if err := printInfoClusterState(awsClient, accessConfig); err != nil {
-		exit.Error(err)
+func cmdInfo(awsClient *aws.Client, accessConfig *clusterconfig.AccessConfig, outputType flags.OutputType, disallowPrompt bool) {
+	if outputType == flags.PrettyOutputType {
+		if err := printInfoClusterState(awsClient, accessConfig); err != nil {
+			exit.Error(err)
+		}
 	}
 
-	clusterConfig := refreshCachedClusterConfig(*awsClient, accessConfig)
+	clusterConfig := refreshCachedClusterConfig(*awsClient, accessConfig, outputType == flags.PrettyOutputType)
 
-	out, exitCode, err := runManagerWithClusterConfig("/root/info.sh", &clusterConfig, awsClient, nil, nil, nil)
+	operatorLoadBalancer, err := getLoadBalancer(accessConfig.ClusterName, OperatorLoadBalancer, awsClient)
 	if err != nil {
 		exit.Error(err)
 	}
-	if exitCode == nil || *exitCode != 0 {
-		exit.Error(ErrorClusterInfo(out))
+	apiLoadBalancer, err := getLoadBalancer(accessConfig.ClusterName, APILoadBalancer, awsClient)
+	if err != nil {
+		exit.Error(err)
 	}
 
-	fmt.Println()
+	operatorEndpoint := *operatorLoadBalancer.DNSName
```

Comment:
nit: `operatorEndpoint := s.EnsurePrefix(*operatorLoadBalancer.DNSName, "https://")` (and remove "https://" below)